routes/drones.js

```diff
@@ -1,36 +1,18 @@
 const express = require('express');
 const router = express.Router();
+const droneController = require('../controllers/droneController')
 
 // require the Drone model here
 
-router.get('/drones', (req, res, next) => {
-  // Iteration #2: List the drones
-  // ... your code here
-});
+router.get('/drones', droneController.getDrones)
```

Comment:
De esta forma en la raiz tendremos el listado 😉 
```suggestion
router.get('/', droneController.getDrones)
```